core/templates/dev/head/components/StaticGraphDirective.js

```diff
@@ -0,0 +1,329 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the state graph visualization.
+ */
+
+oppia.directive('staticGraph', [
```

Comment:
I'm kind of confused here -- what's meant to be the difference between static-graph and state-graph-viz? I still feel like there's too much duplication.